apple/testing/default_runner/ios_sample_test_runner.template.sh

```diff
@@ -57,8 +63,6 @@ XCTEST_PATH="$XCODE_PATH/Platforms/iPhoneSimulator.platform/Developer/Library/Xc
 xcrun simctl spawn "$NEW_SIM_ID" "$XCTEST_PATH" "$TEST_BUNDLE"
 EXIT_CODE=$?
 
-xcrun simctl delete $"$NEW_SIM_ID"
-
 # Bazel detects the exit code from this script as the status of whether the
 # tests succeeded or failed. Any exit code other than 0 means tests failed.
 exit $EXIT_CODE
```

Comment:
I don;t think we need the proxy any more, we can just call `xcrun simctl spawn ...` without storing the exit code... that was used so we had a chance to clean up the sim... but since that is now in trap, we can remove this...